src/app/components/members/Member.js

```diff
@@ -4,17 +4,25 @@ import { Col } from 'react-bootstrap'
 export default class Member extends Component {
   render () {
     const style = { backgroundImage: 'url(' + this.props.img + ')' }
-    const email = 'mailto:' + this.props.mail
-    const twitterLink = 'https://twitter.com/' + this.props.twitter
-    const gitHubLink = 'https://github.com/' + this.props.github
-
+    let mailLink
+    if (typeof this.props.mail !== 'undefined') {
+      mailLink = <a target='_blank' href={'mailto:' + this.props.mail} className='social-icon fa fa-envelope-o' />
+    }
+    let twitterLink
+    if (typeof this.props.twitter !== 'undefined') {
+      twitterLink = <a target='_blank' href={'https://twitter.com/' + this.props.twitter} className='social-icon fa fa-twitter' />
+    }
+    let gitHubLink
+    if (typeof this.props.github !== 'undefined') {
```

Comment:
I apologize if I'm being dumb, but when will `this.props.github` be `undefined` or `null` but still pass the check?

I made this test: 
<img width="226" alt="screen shot 2016-11-22 at 19 35 18" src="https://cloud.githubusercontent.com/assets/6756384/20538917/1fa5d45a-b0eb-11e6-96e1-23096336ba9a.png">

Is there any edge case I'm not considering?
Btw, what do you mean with more reliable and acurate? :)

`0` or `''` will still pass the test